resources/views/models/edit.blade.php

```diff
@@ -34,7 +34,9 @@
 </div>
 
 <!-- Custom Fieldset -->
-@livewire('custom-field-set-default-values-for-model',["model_id" => $item->id])
+@if (isset($item->id))
```

Comment:
Wouldn't this prevent the custom field from loading on a new asset model, since the asset model doesn't have an ID yet since it's not saved yet?

Reply:
it works, i've manually added that code :)